### PR TITLE
Back end development

### PR DESCRIPTION
**Pull request by garretthoffa**
Finished the Delete and Get routes for the backend

### CODE REVIEW

File: backend/routes/postPutRoutes.js [hunk 1]
```diff
@@ -53,11 +55,27 @@ var createTransaction = function(req, res, next) {
...
 };
 
+//This function takes in a budgetID of the budget that you would like to
+//modify and the goal that you would like to set it too. Returns a 200
+//if the goal was updated succesfully
 var changeGoal = function (req, res, next) {
-
+	var budgetRef = db.collection('budgets').doc(req.body.budgetID);
+	
+	cityRef.set({
```
Comment (jnielson94) [inline backend/routes/postPutRoutes.js]: cityRef?

File: backend/routes/postPutRoutes.js [hunk 2]
```diff
@@ -94,18 +94,39 @@ var changeBudgetInfo = function (req, res, next) {
...
+		console.error("Error updating document: ", error);
+		return res.status(500);
 	});
 
 };
 
 var changeAccountInfo = function(req, res, next) {
+	var accountRef = db.collection('budgets').doc(req.body.budgetID).collection('accounts').doc(req.body.accountID);
+
+	var docDate = {};
```
Comment (jnielson94) [inline backend/routes/postPutRoutes.js]: should this be docData?

File: backend/routes/postPutRoutes.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+	});
+};
+
+//This function takes in a budgetID of the budget that you would like to
+//modify and the goal that you would like to set it too. Returns a 200
+//if the goal was updated succesfully
+var changeGoal = function (req, res, next) {
+	var budgetRef = db.collection('budgets').doc(req.body.budgetID);
+	
+	cityRef.set({
```
Comment (jnielson94) [inline backend/routes/postPutRoutes.js]: cityRef?

Comment (garretthoffa) [reply backend/routes/postPutRoutes.js]: just changed will push

File: backend/routes/postPutRoutes.js (hunk 2 above)
Comment (garretthoffa) [reply backend/routes/postPutRoutes.js]: Fixed will push

**Review by jnielson94 — APPROVED** [review]
Most everything seems to be working - though I'm not sure yet about what format dates are going to be in. I'm game to merge this in and we can chat with the front-end guys on Monday and figure out what format they're wanting to work in.

**Pull request merged**